javav2/example_code/acmpca/src/test/java/PCATests.java

```diff
@@ -1,233 +1,246 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import com.example.acmpca.CreateCertificateAuthority;
-import com.example.acmpca.DeleteCertificateAuthority;
-import com.example.acmpca.DescribeCertificateAuthority;
-import com.example.acmpca.ListCertificateAuthorities;
-import com.example.acmpca.GetCertificateAuthorityCertificate;
-import com.example.acmpca.ListTags;
-import com.example.acmpca.GetCertificate;
-import com.example.acmpca.GetCertificateAuthorityCsr;
-import com.example.acmpca.IssueCertificate;
-import com.example.acmpca.TagCertificateAuthorities;
-import com.example.acmpca.UntagCertificateAuthority;
-
+import com.example.acmpca.*;
 import org.junit.jupiter.api.*;
```

Comment:
I figure while you're changing that other thing, fixing star imports could happen here too. Should be easy, just delete the line, see where your compile errors are, then fix them. IDEs should help with this